tests/test_runner.py

```diff
@@ -3,6 +3,10 @@
 from unittest.mock import patch
 import time
 
+from runner import make_volume_name
+from runner import run_cohort_extractor
+from runner.exceptions import BadDockerImageName
```

Comment:
`make_volume_name` and `BadDockerImage` are imported but not used.

Reply:
Bad commit-splitting on my part; they are used in the next commit